resources/cdk/lambda_using_scheduled_events/setup.ts

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env node
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO: Customize this AWS Cloud Development Kit (AWS CDK) code for your
+// specific AWS CDK solution. For more information, see the
+// AWS CDK Developer Guide at
+// https://docs.aws.amazon.com/cdk/latest/guide
+
+import 'source-map-support/register';
+import * as cdk from '@aws-cdk/core';
+import { CfnOutput } from "@aws-cdk/core";
+import * as s3 from '@aws-cdk/aws-s3';
+import * as dynamodb from '@aws-cdk/aws-dynamodb';
+import * as iam from '@aws-cdk/aws-iam';
+import {Effect, PolicyStatement, ServicePrincipal} from '@aws-cdk/aws-iam';
+import { AwsCustomResource } from '@aws-cdk/custom-resources';
+
+export class SetupStackLambda extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+      const myBucket = new s3.Bucket(this, 'mybucket      ',{
+          removalPolicy: cdk.RemovalPolicy.DESTROY,
+          blockPublicAccess:  {
+              blockPublicAcls: true,
+              blockPublicPolicy: false,
+              ignorePublicAcls: true,
+              restrictPublicBuckets: false
+          }
+      });
+      myBucket.grantPublicAccess('*', 's3:GetObject');
+      myBucket.grantPublicAccess('*', 's3:PutObject');
```

Comment:
Does the example require public access or is there a better/more secure way we should be encouraging?

Reply:
fixed